rust/src/measurements/laplace/float/make_scalar_float_laplace_cks20.tex

```diff
@@ -0,0 +1,115 @@
+\documentclass{article}
+\input{../../../lib.sty}
+
+
+
+\title{\texttt{fn make\_scalar\_float\_laplace\_cks20}}
+\author{Michael Shoemate}
+\begin{document}
+
+\maketitle
+
+\contrib
+
+Proves soundness of \rustdoc{measurements/fn}{make\_scalar\_float\_laplace\_cks20} in \asOfCommit{mod.rs}{f5bb719}.
+The function on the resulting measurement takes in a data set \texttt{x} (a single float), 
+and returns a sample from the Discrete Laplace Distribution centered at \texttt{x}, with a fixed noise scale.
+
+\subsection*{PR History}
+\begin{itemize}
+    \item \vettingPR{490}
+\end{itemize}
+
+\section{Hoare Triple}
+
+\subsection*{Preconditions}
+\begin{itemize}
+    \item Variable \texttt{input\_domain}, of type \texttt{AtomDomain<T>}
+    \item Variable \texttt{input\_metric}, of type \texttt{AbsoluteDistance<T>}
+    \item Variable \texttt{scale}, of type \texttt{QO}
+    \item Variable \texttt{k}, of type \texttt{Option<i32>}
```

Comment:
Might be worth defining what this variable $k$ is, i.e., what its purpose is, etc., somewhere before starting the formal proof.